app/views/episodes/new.slim

```diff
@@ -0,0 +1,65 @@
+.container
+  .py-3.text-center
+    img.d-block.mx-auto.mb-4[src="https://i1.sndcdn.com/avatars-000591178137-3dyxzr-t200x200.jpg" alt="proConf logo" width="72" height="72"]
```

Comment:
???? link to constant, styles to css

Reply:
Remove all static links, all dynamic generate via helpers

Reply:
Moved images available on static links to /public. Others are in helper